lib/stat_tracker.rb

```diff
@@ -24,11 +24,11 @@ def initialize(data)
   
   # Game Statistics
   def highest_total_score
-    
+    @games.map {|game| game.away_goals + game.home_goals}.max
   end
 
   def lowest_total_score
-
+    @games.map {|game| game.away_goals + game.home_goals}.min
```

Comment:
nice work, this is clean. it might even be possible to do max_by and min_by here in place of map, meaning we wouldn't have to call max and min at the end. This would also reduce processing power if we cared about that.